### ISSUE TITLE
gphotoframe stops changing pictures, similar to issue 94

### ISSUE THREAD

**Issue by GoogleCodeExporter** [yendo/gphotoframe#110]
```
What steps will reproduce the problem?
1.
2.
3.

What is the expected output? What do you see instead?
A few pictures are changed and then the application stops loading pictures.  
Problem occurs with the default setup.  I've also tried adding / removing 
sources, same problem.

Opening preferences apparently starts pictures changing again.

What version of the product are you using? On what operating system?

 2 alpha Running on Ubuntu 12.04
 1.5.1. was running on Ubuntu 11.04

Both exhibited exactly the same problem.

Please provide any additional information below.

```

Original issue reported on code.google.com by `ronsega...@gmail.com` on 15 Nov 2012 at 9:03

**Comment by GoogleCodeExporter** [comment]
```
I can confirm the same problem. I am running Fedora 18 on a Lenovo T61.
Running 2.0-a2.
```

Original comment by `lilzebr...@gmail.com` on 4 Jul 2014 at 3:51

**Comment by GoogleCodeExporter** [comment]
```
Fixed a related bug in revision 4fb32b74a755.

Could you show me the error messages,
if your problem seems not to be fixed?

```

Original comment by `yendo0206` on 17 Nov 2014 at 4:15
- Changed state: **Fixed**
